### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by diamat1997**
- [DEMO LINK](https://diamat1997.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://diamat1997.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap);
+
+body {
+  font-family: Roboto, sans-serif;
+  padding: 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:5]: body doesn't have default paddings

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(/images/star-active.svg);
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:146]: move each BEM block to the seperate file

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (diamat1997) [reply src/style.css]: Снимок
I did as you said, transferred the code to another file, the styles are pulled up, but now the tests do not pass. If I return the same code, the tests pass.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+}
+
+.card__buy_button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
```
Comment (AlisaVlasova) [inline src/style.css:100]: do not hardcode width, use card paddings to set right spacing, like this
image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  height: 38px;
```
Comment (AlisaVlasova) [inline src/style.css:101]:
```suggestion
  height: 40px;
```
add * {box-sizing: border-box;}

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+  margin-bottom: 16px;
+  text-transform: uppercase;
+  font-style: normal;
```
Comment (AlisaVlasova) [inline src/style.css:104]: this is default

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,121 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap);
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (AlisaVlasova) [inline src/style.css:8]: move it to *

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,121 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap);
+
+body {
+  font-family: Roboto, sans-serif;
```
Comment (AlisaVlasova) [inline src/style.css:4]: move it to html selector, add fallback font

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]
check font-weight or color to fix tests
image

**Pull request closed without merging**